app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:appengine/appengine.dart';
+import 'package:gcloud/db.dart';
+import 'package:meta/meta.dart';
+
+import '../model/appengine/key_helper.dart';
+import '../model/appengine/task.dart';
+import '../model/luci/push_message.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../request_handling/exceptions.dart';
+import '../request_handling/subscription_handler.dart';
+import '../service/config.dart';
+import '../service/datastore.dart';
+import '../service/logging.dart';
+
+/// An endpoint for listening to build updates for postsubmit builds.
+///
+/// The PubSub subscription is set up here:
+/// https://cloud.google.com/cloudpubsub/subscription/detail/luci-postsubmit?project=flutter-dashboard&tab=overview
```

Comment:
Where can I take a look at the messages? Used the url with pantheon, but no messages are available?

Reply:
PubSub messages aren't available in GCP. You can view them in the logs by searching for the handlers: `/api/presubmit-luci-subscription`